server/test/uk/gov/ons/addressIndex/server/modules/ElasticsearchRepositorySpec.scala

```diff
@@ -5393,7 +5432,7 @@ class ElasticsearchRepositorySpec extends WordSpec with SearchMatchers with Clas
       result shouldBe expected
     }
 
-
+   // container.stop()
```

Comment:
Is this comment needed?

Reply:
Probably can go. It is a line that is supposed to be there to tidy up on completion, but tests fail due to timing issues if it is left in (could probably be told explicity to fire on tear down). Docker seems to clean itself OK without it.